src/pages/preferences.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <VContentPage>
+    <h1>{{ $t('pref-page.title') }}</h1>
+
+    <h2>{{ $t('pref-page.toggle') }}</h2>
+    <ul class="!ps-0">
+      <li
+        v-for="(feature, name) in flags"
+        :key="name"
+        class="flex flex-row items-center"
+      >
+        <VCheckbox
+          :id="name"
+          class="flex-row items-center"
+          :checked="featureState(name) === ON"
+          :disabled="feature.status !== SWITCHABLE"
+          @change="handleChange"
+        >
+          <div>
+            <strong>{{ name }}</strong>
+            <br />
+            {{ feature.description }}
+          </div>
+        </VCheckbox>
+      </li>
+    </ul>
+
+    <p>{{ $t('pref-page.save-desc') }}</p>
+    <VButton variant="primary" @click="handleSave">{{
+      $t('pref-page.save')
+    }}</VButton>
+
+    <h2>{{ $t('pref-page.store-state') }}</h2>
+    <pre><code>{{ flags }}</code></pre>
+
+    <h2>{{ $t('pref-page.content-filtering') }}</h2>
+    <template
+      v-for="featName in ['feat_enabled', 'feat_disabled', 'feat_switchable']"
+    >
+      <template v-for="featState in [ON, OFF]">
+        <i18n
+          v-if="featureState(featName) === featState"
+          :key="`${featName}-${featState}`"
+          path="pref-page.explanation"
+          tag="p"
+        >
+          <template #feat-name
+            ><code>{{ featName }}</code></template
+          >
+          <template #feat-state
+            ><code>{{ featState }}</code></template
+          >
+        </i18n>
+      </template>
+    </template>
+  </VContentPage>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, useContext } from '@nuxtjs/composition-api'
+
+import { useFeatureFlagStore } from '~/stores/feature-flag'
+import { SWITCHABLE, ON, OFF } from '~/constants/feature-flag'
+
+import VContentPage from '~/components/VContentPage.vue'
+import VButton from '~/components/VButton.vue'
+import VCheckbox from '~/components/VCheckbox/VCheckbox.vue'
+
+export default defineComponent({
```

Comment:
Should we hide this page in production? It doesn't seem like it should be exposed in prod and it'd be nice to _not_ expose it so that we can ignore things like UI/UX and the expectation that it's anything other than an "internal" development/testing tool.

Reply:
I'm more inclined towards refining the UI and making it a prod-ready page. That way we might be able to do beta-releases of new features and UI changes.

Reply:
> That way we might be able to do beta-releases of new features and UI changes.

How would this work?

Reply:
Beta releases as in features going to production, while still being iterated or improved? If a feature is switchable in production with a [BETA] badge on it, that's essentially possible if we enable this page in prod.

Reply:
Ah, understood!